tooling/ssa_fuzzer/fuzzer/Cargo.toml

```diff
@@ -13,15 +13,21 @@ libfuzzer-sys = { version = "0.4.0", features = ["arbitrary-derive"] }
 noirc_evaluator.workspace = true
 noirc_driver.workspace = true
 acvm.workspace = true
-log.workspace = true
-env_logger.workspace = true
+noir_ssa_executor.workspace = true
+log = "0.4"
+env_logger = "0.11.6"
 
 [dependencies.noir_ssa_fuzzer]
 path = "../"
 
 [[bin]]
 name = "base_target"
```

Comment:
Did you want to leave this without renaming? It's bit confusing, because both files exist.